.circleci/config.yml

```
@@ -36,6 +36,20 @@ restore_dependency_cache_win: &restore_dependency_cache_win
       - v9-cache-win-{{ checksum "package-lock.json" }}
       - v9-cache-win-
 
+# install the version of chromedriver that matches the currently installed version of Chrome (taken from CircleCI chromedriver script)
+install_chromedriver: &install_chromedriver
```

Might be nice to make it explicit that this cannot be used on Windows.

```suggestion
install_linux_chromedriver: &install_linux_chromedriver
```